Sources/Fullscreen/AddressView/AddressView.swift

```diff
@@ -0,0 +1,222 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+public protocol AddressViewDelegate: class {
+    func addressViewDidSelectCopyButton(_ addressView: AddressView)
+    func addressViewDidSelectGetDirectionsButton(_ addressView: AddressView)
+    func addressViewDidSelectCenterMapButton(_ addressView: AddressView)
+    func addressView(_ addressView: AddressView, didSelectMapTypeAtIndex index: Int)
+}
+
+@objc public class AddressView: UIView {
+    public weak var delegate: AddressViewDelegate?
+
+    private lazy var mapTypeSegmentControl: UISegmentedControl = {
+        let control = UISegmentedControl(items: [])
+        control.translatesAutoresizingMaskIntoConstraints = false
+        control.addTarget(self, action: #selector(mapTypeChanged), for: .valueChanged)
+        control.tintColor = .primaryBlue
+        return control
+    }()
+
+    private lazy var segmentContainer: UIView = {
+        let segmentContainer = UIView()
+        segmentContainer.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
You could rather use our FK init, `let segmentContainer = UIView(withAutoLayout: true)` to get rid of this line 🙌